charter.md

```diff
@@ -85,15 +93,123 @@ are replacing, regardless of the length of that remainder.
 
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
```

Comment:
Do we want to specify what "in writing" amounts to? An email to steering-private@ ?

Reply:
IMHO, I don't think that we do.

Reply:
> IMHO, I don't think that we do.

(Agreed. Resolving...)